### PR TITLE
CMP-1877: File Integrity Operator 1.2.0 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/CMP-1877

Link to docs preview:
[OpenShift File Integrity Operator 1.2.0](http://file.rdu.redhat.com/antaylor/CMP-1877/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-1-2-0)
[Creating the FileIntegrity custom resource](http://file.rdu.redhat.com/antaylor/CMP-1877/security/file_integrity_operator/file-integrity-operator-understanding.html#understanding-file-integrity-custom-resource_file-integrity-operator)
[Important attributes](http://file.rdu.redhat.com/antaylor/CMP-1877/security/file_integrity_operator/file-integrity-operator-configuring.html#important-file-integrity-object-attributes_file-integrity-operator)
[Installing the File Integrity Operator using the CLI](http://file.rdu.redhat.com/antaylor/CMP-1877/security/file_integrity_operator/file-integrity-operator-installation.html#installing-file-integrity-operator-using-cli_file-integrity-operator-installation)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mkumku** [comment]
Andrew, I do not see where you mention to use stable channel.

**Comment by Vincent056** [comment]
/lgtm,
Thanks for adding the `initialDelay` documentation.

File: modules/file-integrity-understanding-file-integrity-cr.adoc [hunk 1]
```diff
@@ -23,10 +23,34 @@ metadata:
...
+  debug: false
+status:
+  phase: Active <7>
 ----
+<1> Defines the selector for scheduling node scans.
+<2> Specify `tolerations` to schedule on nodes with custom taints. When not specified, a default toleration allowing running on master and infra nodes is applied.
+<3> Define a `ConfigMap` containing an AIDE configuration to use.
+<4> The number of seconds to pause in between AIDE integrity checks. Frequent AIDE checks on a node may be resource intensive, so it might be useful to specify a longer interval. Default is 900 seconds (15 minutes).
+<5> The maximum number of AIDE database and log backups (leftover from the re-init process) to keep on a node. Older backups beyond this number are automatically pruned by the daemon. Default is set to 5.
+<6> The number of seconds to wait before starting the first AIDE integrity check. Default is set to 0.
```
Comment (xiaojiey) [inline modules/file-integrity-understanding-file-integrity-cr.adoc:48]: @sheriff-rh Could you also add spec.config.initialDelay to important-file-integrity-object-attributes part? Thanks. https://57230--docspreview.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-configuring.html#important-file-integrity-object-attributes_file-integrity-operator

**Comment by sheriff-rh** [comment]
> Andrew, I do not see where you mention to use stable channel.

Updated to mention the `stable` channel. Thanks for the reminder!

**Comment by sheriff-rh** [comment]
> Thanks for the update! Sorry, one more comment, the channel in the subscription should also be updated to 'stable'. 

Great catch! Thanks for your hard work and review efforts! I have updated the pull request.

File: modules/file-integrity-understanding-file-integrity-cr.adoc [hunk 2]
```diff
@@ -23,10 +23,34 @@ metadata:
...
+    key: "config"
+    gracePeriod: 20 <4>
+    maxBackups: 5 <5>
+    initialDelay: 60 <6>
+  debug: false
+status:
+  phase: Active <7>
 ----
+<1> Defines the selector for scheduling node scans.
+<2> Specify `tolerations` to schedule on nodes with custom taints. When not specified, a default toleration allowing running on master and infra nodes is applied.
```
Comment (opayne1) [inline modules/file-integrity-understanding-file-integrity-cr.adoc:44]: is this one of those cases where master is allowed? Or can we use control plane?

**Review by opayne1 — COMMENTED** [review]
Just two small notes for your consideration, otherwise lgtm!

File: modules/file-integrity-understanding-file-integrity-cr.adoc (hunk 2 above)
Comment (sheriff-rh) [reply modules/file-integrity-understanding-file-integrity-cr.adoc:44]: Good catch, I'm still bad at that. 

I think we can use "main" in this instance.

File: modules/file-integrity-understanding-file-integrity-cr.adoc [hunk 3]
```diff
@@ -23,10 +23,34 @@ metadata:
...
+<3> Define a `ConfigMap` containing an AIDE configuration to use.
+<4> The number of seconds to pause in between AIDE integrity checks. Frequent AIDE checks on a node may be resource intensive, so it might be useful to specify a longer interval. Default is 900 seconds (15 minutes).
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply modules/file-integrity-understanding-file-integrity-cr.adoc:46]: That makes so much more sense, thank you!

**Comment by sheriff-rh** [comment]
Release is live. Merging!

**Pull request merged**